src/core/IconButton/index.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import DehazeIcon from "@material-ui/icons/Dehaze";
```

Comment:
As mentioned in the PR description, I'll add the Filter/Info icon svgs in a followup PR, so using an MUI icon for now!